README.md

```diff
@@ -99,9 +81,19 @@ There's an example [systemd service file](https://github.com/restic/rest-server/
 
 ### Docker
 
+Restic works well inside a container, images are [published to Docker Hub](https://hub.docker.com/r/restic/rest-server).
+
+This repository hold the `Dockerfile` of such image, so you may build a fresh one from git via:
+
+    docker build -t restic/rest-server:latest .
+
+Otherwise just pull the current release directly:
+
+    docker pull restic/rest-server:latest
+
 By default, image uses authentication.  To turn it off, set environment variable `DISABLE_AUTHENTICATION` to any value.
 
-Persistent data volume is located to `/data`.
+Note that, **contraty to the defaults** of `restic-server`, the persistent data volume is located to `/data`.
```

Comment:
"contrary" is misspelled :-)

However I think that the original formulation is enough (I mean "Persistent data volume is located to `/data`). Maybe the whole original phase might be written in bold. However that's only my opinion :-D

Reply:
> "contrary" is misspelled :-)

fixed.

> However I think that the original formulation is enough (I mean "Persistent data volume is located to `/data`). Maybe the whole original phase might be written in bold. However that's only my opinion :-D

I wanted to emphasize that the image's behaviour differs from the defaults above. I believe it to be a good practice to make it clear when something overrides a default!